src/main/java/com/Futurecollars/Lekcja4/Zadanie1/ConsoleMessage.java

```diff
@@ -0,0 +1,12 @@
+package com.Futurecollars.Lekcja4.Zadanie1;
+
+public class ConsoleMessage extends Message{
+
+    String getConsoleMessage= "This is Console Message";
```

Comment:
ta zmienna powinna się nazwyać consoleMessage. getConsoleMessage oznacza wykonanie jakiejś czynności (pobrania wiadomości konsolowej) to nazwa dobra dla metody nie dla zmiennej